types.go

```diff
@@ -148,6 +149,22 @@ const (
 	OpcodeStatus = 2
 	OpcodeNotify = 4
 	OpcodeUpdate = 5
+
+)
+
+// Used in ZONEMD https://tools.ietf.org/html/draft-ietf-dnsop-dns-zone-digest-14
+
+const (
+	// ZoneMD Accepted Schemes
+	ZoneMDSchemeReserved = 0
```

Comment:
I don't think we need the reserved constants, they're really just noise.

Reply:
disagree

Reply:
The usable constants were, the reserved ones serve no purpose and have no practical use whatsoever. It's not merely that they take up extra lines, they also appear in the documentation, require long term compatibility and impose a burden on developers understanding the code or documentation.